README.md

```diff
@@ -56,6 +56,10 @@ Configuration
 In order to make shard-awareness work, token aware host selection policy has to be enabled.
 Please make sure that the gocql configuration has `PoolConfig.HostSelectionPolicy` properly set like in the example below. 
 
+Note |
```

Comment:
In GH preview it sort of does not render properly - let's just put the whole thing in bold.

Reply:
Done